### PR TITLE
Setup dockerfile for node v10.1 using alpine:3.7

### PR DESCRIPTION
**Pull request by muchenbit**
Ran the build on both the dockerfile and dockerfile-tests and everything worked okay.

### CODE REVIEW

**Comment by muchenbit** [comment]
The test needs to be ran again after [this pull request is merged.](https://github.com/bandsintown/docker-alpine/pull/7)

**Review by salsa2k — APPROVED** [review]

**Review by msotoperdomo — APPROVED** [review]
LGTM 👍 nice job

**Comment by muchenbit** [comment]
Just waiting on one final review from @salsa2k before merging.

**Review by salsa2k — APPROVED** [review]

**Pull request merged**
